staking/programs/staking/src/lib.rs

```diff
@@ -557,15 +558,84 @@ pub mod staking {
      * The recipient of a transfer can't vote during the epoch of the transfer.
      */
     pub fn accept_split(ctx: Context<AcceptSplit>) -> Result<()> {
-        // TODO : Split vesting schedule between both accounts
+        let config = &ctx.accounts.config;
+        config.check_frozen()?;
```

Comment:
what does this check_frozen do and why is it needed? (leave a comment?)

Reply:
Let's just get rid of this